src/sync/installation.ts

```diff
@@ -240,38 +261,37 @@ export const processInstallation =
 
 			const processor = tasks[task];
 
-			const handleGitHubError = (err) => {
```

Comment:
Moved this method out to make it testable and renamed it to `isRetryableWithSmallerRequest()`